backend/app/views/spree/admin/payments/_list.html.erb

```diff
@@ -22,7 +22,10 @@
   <tbody>
     <% payments.each do |payment| %>
       <tr id="<%= dom_id(payment) %>" data-hook="payments_row" class="payment vertical-middle" data-payment-id="<%= payment.id %>">
-        <td><%= link_to payment.number, spree.admin_order_payment_path(@order, payment) %></td>
+        <td>
+          <%#= tag :i, class: "fa fa-warning red", title: t('spree.risky') if payment.risky? %>
```

Comment:
Is this line commented by error?

Reply:
👆

Reply:
Yes! Thank you, I must have done it to grab the screenshot and then committed with the rest 😓